test/Feature/LargeArrayBecomesSym.c

```diff
@@ -3,7 +3,8 @@
 
 // RUN: %clang %s -emit-llvm %O0opt -c -o %t1.bc
 // RUN: rm -rf %t.klee-out
-// RUN: %klee --exit-on-error --output-dir=%t.klee-out %t1.bc 2>&1 | FileCheck %s
+/* The solver-timeout is needed as some solvers, such as metaSMT+CVC4 which times out here. */
```

Comment:
Can you fix this sentence?
`The solver-timeout is needed as some solvers, such as metaSMT+CVC4, time out here.`
or
`The solver-timeout is needed for some solvers, ...`

Reply:
I guess it was really late :)